### PR TITLE
2021-03-03: lang team march update

### PR DESCRIPTION
**Pull request by nikomatsakis**

### CODE REVIEW

File: posts/inside-rust/2021-03-03-lang-team-mar-update.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    * generally positive vibes about the [RFC 3058](https://github.com/rust-lang/rfcs/pull/3058/)
+    * one question is whether to consider leveraging edition to tweak the desugaring and bypass some of the "accidental stabilization"
+        * currently evaluating how many projects rely on this behavior; most of them don't want to
+* ffi-unwind:
+    * C-unwind implementation has almost landed
+    * exploring impact of longjmp on optimization
+
+## Upcoming design meetings
+
+We planned three design meetings for March. Our meetings are typically open for anyone to observe, although we have one closed meeting this week. They are also typically recorded and posted to YouTube. Ping nikomatsakis or joshtriplett for info about attending.
```
Comment (Mark-Simulacrum) [inline posts/inside-rust/2021-03-03-lang-team-mar-update.md:50]: closed meeting? lang team reorg? (nit: this week -> this cycle)

File: posts/inside-rust/2021-03-03-lang-team-mar-update.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+* March 10 -- no meeting, pnkfelix is absent
+* March 17 -- RFC backlog bonanza recap ([lang-team#84](https://github.com/rust-lang/lang-team/issues/84))
+* March 24 -- lang team reorg: shepherds 4.0, triage update ([lang-team#85](https://github.com/rust-lang/lang-team/issues/85))
+* March 24 -- How to dismantle an `&Atomic` bomb ([lang-team#82](https://github.com/rust-lang/lang-team/issues/82))
+
+## Design meeting expectations
+
+We also settled some our expectations are preparation for design meetings:
```
Comment (Mark-Simulacrum) [inline posts/inside-rust/2021-03-03-lang-team-mar-update.md:59]:
```suggestion
We also settled some our expectations around preparation for design meetings:
```

File: posts/inside-rust/2021-03-03-lang-team-mar-update.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+* The document for the meeting must be prepared by the triage meeting on Tuesday.
+    * If it is not sent out by then, the meeting will be canceled. This gives folks 24 hour notice.
+* There is no expectation that people will read the document before the meeting. The meeting will begin with a recap of the document.
+    * However, there is no rule **against** reading the document before hand and provided feedback or advice on how to improve it.
```
Comment (Mark-Simulacrum) [inline posts/inside-rust/2021-03-03-lang-team-mar-update.md:64]:
```suggestion
    * However, there is no rule **against** reading the document beforehand and provided feedback or advice on how to improve it.
```

File: posts/inside-rust/2021-03-03-lang-team-mar-update.md (hunk 1 above)
Comment (nikomatsakis) [reply posts/inside-rust/2021-03-03-lang-team-mar-update.md:50]: that is some old text, let me edit it out

File: posts/inside-rust/2021-03-03-lang-team-mar-update.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    * exploring impact of longjmp on optimization
+
+## Upcoming design meetings
+
+We planned three design meetings for March. Our meetings are typically open for anyone to observe, although we have one closed meeting this week. They are also typically recorded and posted to YouTube. Ping nikomatsakis or joshtriplett for info about attending.
+
+* March 10 -- no meeting, pnkfelix is absent
+* March 17 -- RFC backlog bonanza recap ([lang-team#84](https://github.com/rust-lang/lang-team/issues/84))
+* March 24 -- lang team reorg: shepherds 4.0, triage update ([lang-team#85](https://github.com/rust-lang/lang-team/issues/85))
+* March 24 -- How to dismantle an `&Atomic` bomb ([lang-team#82](https://github.com/rust-lang/lang-team/issues/82))
```
Comment (jyn514) [inline posts/inside-rust/2021-03-03-lang-team-mar-update.md:55]: I want to say this is an amazing pun :laughing:

Comment (nikomatsakis) [reply posts/inside-rust/2021-03-03-lang-team-mar-update.md:55]: credit to @pnkfelix

**Comment by nikomatsakis** [comment]
@Mark-Simulacrum did you want to review this?

**Pull request merged**